tests/models/musicgen/test_modeling_musicgen.py

```diff
@@ -452,7 +452,6 @@ def test_flash_attn_2_inference_equivalence_right_padding(self):
 
     @parameterized.expand([("float16",), ("bfloat16",), ("float32",)])
     @require_torch_sdpa
-    @slow
```

Comment:
Can't we just delete the test? (it has `# Copied from tests.test_modeling_common.ModelTesterMixin.test_eager_matches_sdpa_inference` and it inherits the mixin, so it should run the original test!)

Reply:
no, it fails. I don't dive into why it's failing (input issues) though.

Reply:
Interesting -- in that case, how does `# Copied from ` work? 👀